library/src/lib/calendar/calendar-views/calendar-year-view/calendar-year-view.component.ts

```diff
@@ -136,4 +149,13 @@ export class CalendarYearViewComponent implements AfterViewChecked, OnInit, OnDe
         this.yearSelected = selectedYear;
         this.yearClicked.emit(this.yearSelected);
     }
+
+    /** @hidden */
+    private constructYearList(): void {
+        this.calendarYearList = [];
+        for (let x = 0; x < 12; x++) {
```

Comment:
Put the 12 in a const above the loop and call it something like "displayedYearsAmount" or something

Reply:
also use `++x` instead of `x++` pls

Reply:
I'm not sure what is the reason, for changing `x++ to `++x`, but it's fine.

Reply:
The reason is my OCD, and my teachers always used ++x so I'm used to it, and we might as well have the same everywhere